checker/tests/interning/Issue2809.java

```diff
@@ -0,0 +1,25 @@
+// test cases for #2809
+// https://github.com/typetools/checker-framework/issues/2809
+
+import org.checkerframework.checker.interning.qual.Interned;
+
+class Issue2809 {
```

Comment:
Could you add an expected failure test, to show that things that should go wrong still cause an error?

Reply:
Do you mean a false negative one or just a normal expected failure?